### PR TITLE
Add a link to SideCI into the documentation

### PR DESCRIPTION
**Pull request by pocke**
I work at Actcat inc. and we develop SideCI in this company. https://sideci.com
This service is similar to Codacy and Hound.

I added SideCI section into rubocop documentation.

### CODE REVIEW

File: manual/automated_code_review.md [hunk 1]
```diff
@@ -12,4 +12,8 @@ Codacy is free for open source, and it provides RuboCop analysis out-of-the-box.
 
 ### Hound
 
-[Hound](https://github.com/houndci/hound) is yet another alternative, but their main focus is code style.
\ No newline at end of file
+[Hound](https://github.com/houndci/hound) is yet another alternative, but their main focus is code style.
+
+### SideCI
+
+[SideCI](https://sideci.com) is yet another alternative. It improves your team's productivity by automating code analysis.
```
Comment (bbatsov) [inline manual/automated_code_review.md]: I'd use some other wording here. The current text was written by someone who cares about `codacy` only - that's why the other services have pretty much no description.

**Comment by bbatsov** [comment]
Nice! It's good to see your company investing time (your time) to help with RuboCop's development.

File: manual/automated_code_review.md (hunk 1 above)
Comment (pocke) [inline manual/automated_code_review.md]: 👍 I'll add more descriptions for the other services.

**Comment by pocke** [comment]
@bbatsov  I updated the descriptions for other services.

**Pull request merged**